Makefile

```diff
@@ -143,9 +143,8 @@ install: default
 	$(INSTALL) -d /usr/local/apisix/
 	$(INSTALL) -d /usr/local/apisix/logs/
 	$(INSTALL) -d /usr/local/apisix/conf/cert
-	$(INSTALL) conf/mime.types /usr/local/apisix/conf/mime.types
-	$(INSTALL) conf/config.yaml /usr/local/apisix/conf/config.yaml
-	$(INSTALL) conf/config-default.yaml /usr/local/apisix/conf/config-default.yaml
+	$(INSTALL) conf/*.types /usr/local/apisix/conf/
+	$(INSTALL) conf/*.yaml /usr/local/apisix/conf/
```

Comment:
IMHO, it is better to specify the files explicitly to avoid mistake.

Reply:
Thanks for the suggestion! Since we need everything under `/conf`, as what we did when creating tarball, I think it's fine to use `*`. 
Besides, in `make install`, it seems people intend to this kind of style.

Reply:
Unlike the Lua code, the configuration file are independent and should be list explicitly.

Reply:
Changed